### ISSUE TITLE
`ERROR: Syntax error (unterminated string)` with 2.4.1

### ISSUE THREAD

**Issue by alexjfisher** [rodjek/puppet-lint#899]
puppet-lint has started failing in `puppet/extlib`.
```puppet
function extlib::file_separator() >> String {
  ($::facts['kernel'] == 'windows' ) ? { true => "\\", false => '/' }
}
```
```
puppet-lint functions/file_separator.pp 
ERROR: Syntax error (unterminated string) on line 2
```

**Comment by alexjfisher** [comment]
I've tested https://github.com/rodjek/puppet-lint/pull/892 and it fixes this issue.

**Comment by rodjek** [comment]
Fixed by #892
